lib/maze/driver/appium.rb

```diff
@@ -112,6 +112,17 @@ def set_rotation(orientation)
         @driver.rotation = orientation
       end
 
+      # Send keys to the device without a specific element
+      #
+      # @param text [String] the text to send
+      def send_keys(text)
+        @driver.send_keys(text)
+      end
+
+      def hide_keyboard
```

Comment:
nit: add a comment above the method

Reply:
i remoed this method, i don't need it in the end